samples/client/petstore/cpp-restsdk/client/include/CppRestPetstoreClient/model/CreateUserOrPet_request.h

```diff
@@ -0,0 +1,98 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ *
+ * NOTE: This class is auto generated by OpenAPI-Generator 7.7.0-SNAPSHOT.
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+/*
+ * CreateUserOrPet_request.h
+ *
+ * 
+ */
+
+#ifndef ORG_OPENAPITOOLS_CLIENT_MODEL_CreateUserOrPet_request_H_
+#define ORG_OPENAPITOOLS_CLIENT_MODEL_CreateUserOrPet_request_H_
+
+
+#include "CppRestPetstoreClient/ModelBase.h"
+
+#include "CppRestPetstoreClient/model/User.h"
+#include "CppRestPetstoreClient/model/Tag.h"
+#include "CppRestPetstoreClient/model/Category.h"
+#include <cpprest/details/basic_types.h>
+#include "CppRestPetstoreClient/model/Pet.h"
+#include <vector>
+
+namespace org {
+namespace openapitools {
+namespace client {
+namespace model {
+
+class Category;
+class Tag;
+
+#include <variant>
```

Comment:
This is the problem. The include is being rendered inside the namespace instead of top-level